src/plumber.R

```diff
@@ -50,6 +51,20 @@ function(req, res) {
   plumber::forward()
 }
 
+#* Cross-Origin Resource Sharing (CORS) workaround.
+#*
+#* Copied from
+#* https://www.rplumber.io/articles/security.html#cross-origin-resource-sharing-cors
+#*
+#* Enable Cross-Origin Resource Sharing (CORS) by default for all endpoints. To
+#* disable for certain endpoints, add preempt annotation to exclude this filter.
+#*
+#* @filter cors
+cors <- function(res) {
+  res$setHeader("Access-Control-Allow-Origin", "*")
+  plumber::forward()
+}
+
 #* Get a single-gene single-disease all-GTEx-tissue-subgroups TPM summary table
 #*
 #* @tag "Bulk tissue gene expression"
```

Comment:
Regarding your @params, I'd change `a single character value of xyz` to something like, `one xyz`. We know it's supposed to be a string because of `@param xyz:str`. A `single character` is just that - a single character, e.g. `"a"` or `"3"`. 

If you can, i'd add an example of

Reply:
I agree. Implemented in <https://github.com/PediatricOpenTargets/OpenPedCan-api/pull/20/commits/47cc4cae7e5fb48bbc949675a4e22c60cc11a837>.

The `a single character value of` uses R semantics, but I agree `one` is the correct description for API users.